daprdocs/content/en/operations/resiliency/configure-policies.md

```diff
@@ -0,0 +1,282 @@
+---
+type: docs
+title: "How-To: Error recovery using resiliency policies"
+linkTitle: "Resiliency Policies"
+weight: 4500
+description: "Configure Dapr error retries, timeouts and circuit breakers"
+---
+
+Resiliency is currently a preview feature. Before you can utilize resiliency policies you must first [enable the resiliency preview feature]({{<ref preview-features >}}).
+
+## Introduction
+
+- TODO: What is resiliency in Dapr?
```

Comment:
Maybe we could add something along the lines of:

Cloud-native applications are commonly comprised of many moving pieces - there could be dozens or even hundreds of instances for any given service. With this many moving pieces, the likelihood of a system failure increases. An instance can fail for any number of reasons, for example, hardware failures, overwhelming number of requests, application restarts/scale outs. Any of these events can cause a network call between services to fail. Having your application designed with the ability to detect and mitigate these failures allows for your application to  respond and recover quickly back to a functioning state.